resources/views/parents/edit.blade.php

```diff
@@ -0,0 +1,72 @@
+<x-app-layout>
+    <x-slot name="header">
+        <h2 class="font-semibold text-xl text-gray-800 leading-tight">
+            {{ __('Edit Parent') }}
+        </h2>
+    </x-slot>
+
+    <div class="py-12">
+        <div class="max-w-7xl mx-auto sm:px-6 lg:px-8">
+            <div class="bg-white overflow-hidden shadow-xl sm:rounded-lg">
+                <form method="POST" action="{{ route('parents.update', $parent->id) }}" enctype="multipart/form-data">
```

Comment:
Houd de actions zo duidelijk mogelijk en vergeet niet dat dit te zien is in de url. Er is geen reden om in de route de id van de parent mee te geven (tenminste naar mijn weten), wil je de id in de controller gebruiken dan kan je dat via een hidden inputfield meegeven.